Src/DependencyCollector/Shared/Implementation/FrameworkHttpProcessing.cs

```diff
@@ -54,6 +54,14 @@ public void OnBeginHttpCallback(long id, string resourceName)
                     return;
                 }
 
+                var url = new Uri(resourceName);
```

Comment:
I believe this will throw an exception rather than return null which means the check later won't work and this should be wrapped in a try catch.